test/fail_compilation/test21651.d

```diff
@@ -0,0 +1,7 @@
+// EXTRA_SOURCES: imports/test21651a.d imports/test21651b.d
+/* TEST_OUTPUT:
+---
+fail_compilation/imports/test21651a.d(4): Deprecation: module imports.test21651b is not accessible here, perhaps add 'static import imports.test21651b;'
+fail_compilation/imports/test21651a.d(5): Error: undefined identifier `test21651b` in package `imports`, perhaps add `static import imports.test21651b;`
```

Comment:
This error is not necessary as it is covered already by other tests. Simply compile with `-de` and keep the minimum required lines of code to reproduce the code that this PR actually fixes.

Reply:
Done.